lib/AbstractMethodError.js

```diff
@@ -13,7 +13,7 @@ const CURRENT_METHOD_REGEXP = /at ([a-zA-Z0-9_.]*)/;
  * @returns {string} message
  */
 function createMessage(method) {
-	return `Abstract method${method ? " " + method : ""}. Must be overridden.`;
+	return `Abstract method${method ? " " + method : " "}. Must be overridden.`;
```

Comment:
Why should there be a space before `.`?

This results in `Abstract method . Must be overriden.`

Reply:
The test was failing without this, I thought it might be acceptable but the error message would look a bit weird with this. I can look for a way to make it pass without the space.